sound-output-device-chooser@kgshank.net/convenience.js

```diff
@@ -108,7 +108,12 @@ function refreshCards() {
 }
 
 function parseOutput(out) {
-    let lines = ByteArray.toString(out).split('\n');
+    if (out instanceof Uint8Array) {
+        let lines = ByteArray.toString(out).split('\n');
```

Comment:
`let lines` is defined at the block level, not the function level, so `lines` will not be available outside this block.